### PR TITLE
Added debug option

### PR DESCRIPTION
**Pull request by PavelLoparev**
1. Pass debug option to http client (dump curl debug output)
2. Dump request body

all goes to stdout

### CODE REVIEW

**Review by vsolovei-smartling — COMMENTED** [review]
LGTM

File: tests/unit/AuditLogApiTest.php [hunk 1]
```diff
@@ -126,7 +126,7 @@ public function testCreateAccountLevelLogRecord()
                 ],
                 'exceptions' => false,
                 'json' => [
-                    'actionTime' => '2009-02-13T23:31:30Z',
+                    'actionTime' => '2009-02-14T01:31:30Z',
```
Comment (vsolovei-smartling) [inline tests/unit/AuditLogApiTest.php]: Just curious, why?

File: src/BaseApiAbstract.php [hunk 2]
```diff
@@ -462,6 +464,20 @@ function ($a) {
...
     protected function sendRequest($uri, array $options, $method, $returnRawResponseBody = false)
     {
+        $config = $this->getHttpClient()->getConfig();
+        $debug = $config["debug"];
+        $handlerStack = $config["handler"];
+
+        if ($debug) {
+            $handlerStack->push(
+                Middleware::mapRequest(function(RequestInterface $request) {
+                    echo var_export((string) $request->getBody(), true) . PHP_EOL;
```
Comment (vsolovei-smartling) [inline src/BaseApiAbstract.php:474]: I'm assuming this is not an error and will get buffered somewhere

**Review by vsolovei-smartling — APPROVED** [review]

File: tests/unit/AuditLogApiTest.php (hunk 1 above)
Comment (PavelLoparev) [reply tests/unit/AuditLogApiTest.php]: We used `date` and it doesn't convert to UTC, it takes into consideration daylight shifts. Now we use `gmdate` which returns Greenwich time (GMT+0 = UTC)

File: src/BaseApiAbstract.php (hunk 2 above)
Comment (PavelLoparev) [reply src/BaseApiAbstract.php:474]: You mean dumping to stdout? it's by intention

**Comment by dimitrystd** [comment]
Close without merge. The original issue was resolved.

**Pull request closed without merging**